util/features.py

```diff
@@ -218,31 +244,46 @@ def design_features(
     Returns:
       pd.DataFrame of features
     """
-    # for soil, we need to comput water holding capacity
+    # for soil, we need to compute water holding capacity
     # TODO: 1. not needed for cybench data. Remove later.
     # TODO: 2. Add code to make drainage class a categorical feature.
+
+# Not using drainage class feature for the neurips submission
+    # if 'DRAIN' in soil_df.columns:
+    #     soil_df['DRAIN'] = soil_df['DRAIN'].astype('category')
+
     if "sm_whc" not in soil_df.columns:
         soil_df["sm_whc"] = soil_df["sm_fc"] - soil_df["sm_wp"]
         soil_features = soil_df[[KEY_LOC, "sm_whc"]]
     else:
         soil_features = soil_df[[KEY_LOC, "sm_whc"]]
 
     # Feature design for time series
-    # TODO: 1. add code for cumulative features
-    # TODO: 2. add code for ET0, ndvi, soil moisture
     index_cols = [KEY_LOC, KEY_YEAR]
     period_length = "month"
-    max_feature_cols = ["fapar"]  # ["ndvi", "fapar"]
-    avg_feature_cols = ["tmin", "tmax"]  # , "tmax", "tavg", "prec", "rad"]
+    max_feature_cols = ["ndvi", "fapar", "cum_gdd", "cum_cwb", "cum_ndvi", "cum_prec"]
```

Comment:
They need to be separated into 3.
1. weather max
2. fapar max
3. NDVI max
These 3 data sources have to be handled separately. They have different time resolution.